c2cgeoportal/lib/email_.py

```diff
@@ -34,23 +34,23 @@
 from email.Utils import formatdate
 
 
-def send_email(from_addr, to_addrs, body, subject, smtp):  # pragma: no cover
+def send_email(email_config, smtp_config):  # pragma: no cover
     msg = MIMEMultipart()
-    msg["From"] = from_addr
-    msg["To"] = ", ".join(to_addrs)
+    msg["From"] = email_config.get("from_addr")
+    msg["To"] = ", ".join(email_config.get("to_addrs"))
     msg["Date"] = formatdate(localtime=True)
-    msg["Subject"] = subject
-    msg.attach(MIMEText(body, "plain", "utf-8"))
+    msg["Subject"] = email_config.get("subject")
+    msg.attach(MIMEText(email_config.get("body"), "plain", "utf-8"))
 
     # Connect to server
-    if smtp.get("ssl", False):
-        smtp = smtplib.SMTP_SSL(smtp["host"])
+    if smtp_config.get("ssl", False):
+        smtp = smtplib.SMTP_SSL(smtp_config.get("host"))
```

Comment:
The host is optional?

Reply:
it is not,
how would you like me to write that ?